extensions/mssql/src/objectManagement/commands.ts

```diff
@@ -237,6 +241,35 @@ async function handleRenameObjectCommand(context: azdata.ObjectExplorerContext,
 	});
 }
 
+async function handleDetachDatabase(context: azdata.ObjectExplorerContext, service: IObjectManagementService): Promise<void> {
+	const connectionUri = await getConnectionUri(context);
+	if (!connectionUri) {
```

Comment:
we should surface the error, otherwise the user would think it is not doing anything or hanging.

Reply:
Looks like we already show an error notification as part of getConnectionUri, but we don't throw an Error.

Reply:
you are right, this is good enough.